### ISSUE TITLE
support save as jxl file.

### ISSUE THREAD

**Issue by lazyzyf** [jtheoof/swappy#184]

**Comment by voronind-com** [comment]
I agree with @kadamski even tho I am a fan of jxl. Unless there's built-in support in an upstream library there's no need to implement it as it can be easily piped to cjxl.
